provider/lxd/credentials.go

```diff
@@ -81,19 +81,19 @@ func (environProviderCredentials) CredentialSchemas() map[cloud.AuthType]cloud.C
 			{
 				Name: credAttrServerCert,
 				CredentialAttr: cloud.CredentialAttr{
-					Description:    "The LXD server certificate, PEM-encoded.",
+					Description:    "the LXD server certificate filename, PEM-encoded",
```

Comment:
After thinking on this more I guess it's a file path. So what about

    the path to the PEM-encoded LXD server certificate file

Does that read smoother?

Reply:
updated